patterns/pattern-1/images/mysql-test-node/files/metrics.sql

```diff
@@ -0,0 +1,92 @@
+-- 
```

Comment:
This will be fixed later by creating images to start our servers with -Dsetup option.